internal/docstore/memdocstore/mem.go

```diff
@@ -123,7 +130,22 @@ func (c *collection) ErrorCode(err error) gcerr.ErrorCode {
 // RunActions implements driver.RunActions.
 func (c *collection) RunActions(ctx context.Context, actions []*driver.Action, unordered bool) driver.ActionListError {
 	if unordered {
-		panic("unordered unimplemented")
+		errs := make([]error, len(actions))
+		for i, a := range actions {
+			i := i
+			a := a
+			go func() { errs[i] = c.runAction(ctx, a) }()
+		}
```

Comment:
What prevents this from not waiting for all the goroutines to finish?

Reply:
Oops... I'm working a little too fast. Done.

Reply:
As was your code...